domains/certificates/RegistryConnector/Worker/RoutingSlips/SendToWalletActivity.cs

```diff
@@ -23,14 +25,15 @@ public SendToWalletActivity(ILogger<SendToWalletActivity> logger)
     public async Task<ExecutionResult> Execute(ExecuteContext<SendToWalletArguments> context)
     {
         logger.LogInformation("Sending slice to Wallet for certificate id {certificateId}. TrackingNumber: {trackingNumber}",
-            context.Arguments.ReceiveRequest.CertificateId.StreamId.Value, context.TrackingNumber);
+            context.Arguments.ReceiveRequest.CertificateId.StreamId, context.TrackingNumber);
 
-        using var channel = GrpcChannel.ForAddress(context.Arguments.WalletUrl);
-        var client = new ReceiveSliceService.ReceiveSliceServiceClient(channel);
+        var client = new HttpClient();
```

Comment:
Dispose, or better yet inject in constructor

Reply:
Corrected - see https://github.com/Energinet-DataHub/energy-origin/pull/1054